internal/postgres/reconcile.go

```diff
@@ -44,13 +44,21 @@ func InitCopyReplicationTLS(postgresCluster *v1beta1.PostgresCluster,
 		naming.ReplicationTmp, naming.CertMountPath+naming.ReplicationDirectory,
 		naming.ReplicationCert, naming.ReplicationPrivateKey,
 		naming.ReplicationCACert, naming.ReplicationTmp)
-	template.Spec.InitContainers = append(template.Spec.InitContainers,
-		v1.Container{
-			Command:         []string{"bash", "-c", cmd},
-			Image:           config.PostgresContainerImage(postgresCluster),
-			Name:            naming.ContainerClientCertInit,
-			SecurityContext: initialize.RestrictedSecurityContext(),
-		})
+
+	container := v1.Container{
+		Command:         []string{"bash", "-c", cmd},
+		Image:           config.PostgresContainerImage(postgresCluster),
+		Name:            naming.ContainerClientCertInit,
+		SecurityContext: initialize.RestrictedSecurityContext(),
+	}
+
+	// set the NSS wrapper container resources to the 'database' container settings
```

Comment:
I've reworded the comment since it is confusing. The replication cert container will use the same resource configuration settings as the `database` container rather than having a its own `resources` section.